build.sh

```diff
@@ -14,6 +14,35 @@
 # limitations under the License.
 # ****************************************************************************
 
+set -ex
+
+# Install Mono:
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get -y install gnupg ca-certificates
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+echo "deb https://download.mono-project.com/repo/ubuntu stable-bionic/snapshots/6.0.0.319 main" | tee /etc/apt/sources.list.d/mono-official-stable.list
+apt update
+DEBIAN_FRONTEND=noninteractive apt -y install mono-devel=6.0.0.*
```

Comment:
It just occurred to me that we may be able to forego this whole installation orgy by simply `alias`-ing `mono` to something like `docker run --rm mono:<VERSION>`. Not having to install anything would speed up the CI build a lot. I'll try to refactor...